lib/static/components/section/body/index.js

```diff
@@ -78,6 +78,16 @@ class Body extends Component {
         this.props.actions.toggleStateResult({browserId, suitePath, stateName, retryIndex, opened});
     }
 
+    getExtraMetaInfo = () => {
+        const {suite, apiValues: {extraItems, metaInfoExtenders}} = this.props;
+
+        return omitBy(mapValues(metaInfoExtenders, (extender) => {
+            const stringifiedFn = extender.startsWith('return') ? extender : `return ${extender}`;
+
+            return new Function(stringifiedFn)()(suite, extraItems);
```

Comment:
how it works? You'll pass some 'bla' as a value for extender and here it will be `function() { return 'bla'}()(suite, extraItems)`?

Reply:
I describe how it works in my previous comment. And it will work only with functions. I fix docs and now extender can only be a function.